game.py

```diff
@@ -64,19 +66,29 @@ def put_down_card_on_table(self, player, card):
 
     def try_playing_matching_number_or_color_card(self, player):
         for card in player.cards:
-            if card.color == self.card_on_table.color or card.num == self.card_on_table.num:
+            if (card.color == self.card_on_table.color or card.num == self.card_on_table.num) and \
```

Comment:
maybe Card could have a method which would say if its a match with a different card and can be set on it? (i.e. it'd do what you do here but it'd be the property of the card)

Reply:
Really like that 🤩 
Great OOP approach :D